### PR TITLE
constants.xml Unify titles for families of constants

### PR DESCRIPTION
**Pull request by mmalferov**

### CODE REVIEW

File: reference/imagick/constants.xml [hunk 1]
```diff
@@ -121,7 +121,7 @@
  </variablelist>
 
  <variablelist xml:id="imagick.constants.disposetype">
-  <title>DISPOSE constants</title>
+  <title>DISPOSE_* constants</title>
```
Comment (cmb69) [inline reference/imagick/constants.xml:124]: I'm not sure if we should go "all in", i.e.

```suggestion
  <title><constant>DISPOSE_<replaceable>*</replaceable></constant> constants</title>
```

What do you think?

Comment (sy-records) [reply reference/imagick/constants.xml]: 🤔️ Others in this PR should also add `<constant> <replaceable>*</replaceable></constant>`

Comment (mmalferov) [reply reference/imagick/constants.xml]: You're right, it's my mistake. It's done! 👍

Comment (sy-records) [reply reference/imagick/constants.xml]: missing constant

Comment (mmalferov) [reply reference/imagick/constants.xml]: @sy-records Could you please show me where the constants are missing?

File: reference/imagick/constants.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
  &reftitle.constants;
  &extension.constants;
  <variablelist xml:id="imagick.constants.colortype">
-  <title>COLOR_* constants</title>
+  <title>COLOR_<replaceable>*</replaceable> constants</title>
```
Comment (sy-records) [inline reference/imagick/constants.xml:7]:
```suggestion
  <title><constant>COLOR_<replaceable>*</replaceable></constant> constants</title>
  
```

Comment (mmalferov) [reply reference/imagick/constants.xml]: I don't think I got enough sleep today )) Done!

**Review by sy-records — APPROVED** [review]
Thank you!

**Pull request merged**